src/main/java/frc/robot/subsystems/WristSubsystem.java

```diff
@@ -8,147 +8,51 @@
 import com.revrobotics.CANSparkMax.IdleMode;
```

Comment:
Same idea in this file.  If there's a chance that we'll eventually use rotating wrist, it's fine to leave that code in here and not wire it into robotcontainer.